jest.config.ts

```diff
@@ -77,7 +77,7 @@ const config: Config = {
   transformIgnorePatterns: ['node_modules/(?!.*.mjs$)'],
   preset: './jest.preset.js',
   moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageDirectory: './coverage/exasol-driver',
+  coverageDirectory: './coverage',
```

Comment:
is there a reason for the change?

Reply:
Yes, there's no need for the sub-dir. `./coverage/` is empty anyway.
Coverage was not generated before at all.